src/components/Physics.vue

```diff
@@ -57,11 +62,31 @@ onBeforeRender(() => {
     const source2 = getSourceFromColliderHandle(world, handle2)
     const group1 = get3DGroupFromSource(source1, scene)
     const group2 = get3DGroupFromSource(source2, scene)
-    if (group1 && group2) {
-      collisionEmisor(
-        { object: group1, context: source1 },
+
+    if (!group1 || !group2) {
+      return
+    }
+
+    collisionEmisor(
+      { object: group1, context: source1 },
+      { object: group2, context: source2 },
+      started,
+    )
+
+    if (started) {
```

Comment:
A small improvement here.
I know this is the approach of R3F but if you pay attention to my `collisionEmisor` I handle the state (started true or false) in the function itself, this way we don't have to put an extra if in Physic and also don't have to repeat the `emitIntersection` function. Also keep consistency in the code

Reply:
Yes, but it was also my design choice to be able to trigger the starting and the ending.
But yeah, let me correct this

Reply:
I made the change @JaimeTorrealba